src/microbe_stage/systems/MicrobeEmissionSystem.cs

```diff
@@ -285,6 +287,23 @@ private float ToxinCooldownWithToxicity(int vacuoleCount, float toxicity)
         return Constants.AGENT_EMISSION_COOLDOWN / vacuoleCount;
     }
 
+    private float EmissionAmountWithToxicity(float toxicity)
+    {
+        if (toxicity < 0)
+        {
+            // High-firerate, low emission amount
+            return Constants.MAXIMUM_AGENT_EMISSION_AMOUNT * (1.0f + toxicity * 0.5f);
```

Comment:
Doesn't this also increase the amount? Shouldn't there be like 1 **-** toxicity?

Reply:
Oh yeah, I missed that part, it is negative here so subtracting it wouldn't be the right thing to do.

I think this is fine. So this makes the technical side all fine. Let's wait a bit and see if we get any game designer or tester to comment on the balance change of this.